engine/table/src/main/java/io/deephaven/engine/table/impl/updateby/emstd/BigDecimalEmStdOperator.java

```diff
@@ -21,7 +21,7 @@
  * Compute an exponential moving standard deviation for a BigDecimal column source. The output is expressed as a
  * BigDecimal value and is computed using the following formula:
  *
- * variance = alpha * (prevVariance + (1 − alpha) * (x − prevEma)^2)
+ * variance = alpha * (prevVariance + (1 - alpha) * (x - prevEma)^2)
```

Comment:
I can't tell what changed here. Was it some special char?

Reply:
Yes, an em dash to an en dash. The first isn't in ascii, the second is, and prevents copying this code to run it, or irritating cases where the machine doing the building doesnt do utf-8.